doc/manuals/admin/cli.md

```diff
@@ -166,6 +166,7 @@ The list of available options is the following:
 -   **-disableCustomNotifications**. Disabled NGSIv2 custom notifications. In particular:
     * `httpCustom` is interpreted as `http`, i.e. all sub-fields except `url` are ignored
     * No `${...}` macro substitution is performed.
+-   **-disableFileLog**. To prevent orion from logging into a file. This option might be usefull if you are running on kubernetes.
```

Comment:
```suggestion
-   **-disableFileLog**. To prevent Orion from logging into a file (default behaviour is to use a log file). This option might be useful if you are running on kubernetes.
```

Reply:
Fixed in ce77dfe455713e7b52992d067daebfad21136745